components/badge/README.md

```diff
@@ -1,51 +1,23 @@
 # Badges
 
-Badges are a lightweight method of annotating or labeling content. Wrap content in the `.badge` class to add an inline badge. The font-size and line-height of a badge inherit from that of it's parent's properties.
+Badges are a lightweight method of annotating or labeling content. Wrap content in the .badge class to add an inline badge. The font-size and line-height of a badge inherit from that of it’s parent’s properties.
```

Comment:
Here we are talking about a class, but probably we want to mention a component.

Reply:
Agree. I have copied the docs Lee provided to us I will talk to him so we are aligned.